about.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wilsan's First Capstone project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Lato:ital,wght@0,300;0,700;1,700&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <nav>
+        <div class="desktop-nav">
+            <div class="upper-nav">
+                <div class="iconCan d-flex">
+                    <a href="#"><img src="Images/facebook.png" alt=""></a>
+                    <a href="#"><img src="Images/twitter.svg" alt=""></a>
+
+                    <a href="#">English</a>
+                    <a href="#">My page</a>
+                    <a href="#">Logout</a>
+                </div>
+            </div>
+
+            <div class="main-nav">
+                <div class="subNav d-flex">
+                    <div class="subLeft">
+                        <img src="./Images/logo.jpg" alt="">
+                    </div>
+                    <!-- hamburger -->
+                    <div class="ham" id="menu">
+                        <img src="./Images/hamburger.svg" alt="">
+                    </div>
+                    <!-- moblie nav class moblie abso -->
+                    <div class="moblie-abso" id="item">
+                        <!-- close -->
+                        <div class="closes" id="close">
+                            <img src="./Images/close.svg" alt="">
+                        </div>
+                        <div class="subMiddle d-flex">
+                            <a href="index.html" class="active">Home</a>
+                            <a id="li" href="#prog">Program</a>
+                            <a href="">Join</a>
+                            <a href="">Sponsor</a>
+                            <a href="">News</a>
+                        </div>
+                        <div class="subRight">
+                            <a href="#">
+                                CC campaign
+                            </a>
+                        </div>
+                    </div>
```

Comment:
- [ ] [OPTIIONAL] On desktop view there seems to be no navigation to go back to the home page, I kindly suggest you  duplicate the home nav section to the about us